Upstream-Status: Backport

From ed6e297972318a0070ad4d973401fbc6e0def558 Mon Sep 17 00:00:00 2001
From: Serj Kalichev <serj.kalichev@gmail.com>
Date: Fri, 7 Sep 2012 13:29:42 +0400
Subject: [PATCH] build: fix extract.pl for cross compilation

extract.pl should invoke the C preprocessor for the target system, not the
host.

* vtysh/extract.pl.in: use @CPP@ to get target cpp
---
 vtysh/extract.pl.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vtysh/extract.pl.in b/vtysh/extract.pl.in
index 7612aff..4c3a47f 100755
--- a/vtysh/extract.pl.in
+++ b/vtysh/extract.pl.in
@@ -63,7 +63,7 @@ $ignore{'"show history"'} = "ignore";
 foreach (@ARGV) {
     $file = $_;
 
-    open (FH, "cpp -DHAVE_CONFIG_H -DVTYSH_EXTRACT_PL -DHAVE_IPV6 -I@top_builddir@ -I@srcdir@/ -I@srcdir@/.. -I@top_srcdir@/lib -I@top_srcdir@/isisd/topology @SNMP_INCLUDES@ @CPPFLAGS@ $file |");
+    open (FH, "@CPP@ -DHAVE_CONFIG_H -DVTYSH_EXTRACT_PL -DHAVE_IPV6 -I@top_builddir@ -I@srcdir@/ -I@srcdir@/.. -I@top_srcdir@/lib -I@top_srcdir@/isisd/topology @SNMP_INCLUDES@ @CPPFLAGS@ $file |");
     local $/; undef $/;
     $line = <FH>;
     close (FH);
-- 
1.7.1

